### PR TITLE
Initiate document for Approval decision utils

### PR DESCRIPTION
**Pull request by nntthuy-axonivy**

### CODE REVIEW

**Comment by nntthuy-axonivy** [comment]
Hi @andreasbalsiger, I created this pull request for the documentation step.
Could you help review this README.md file, it describes the purposes and implementation of Approval decision utils.
Thanks!

**Comment by ivy-sgi** [comment]
Hi @nntthuy-axonivy !
For the publishing of a marketplace item, it is important that the structure specified in the README.md template https://github.com/axonivy-market/market-product/blob/master/MY-PRODUCT-NAME-product/README.md is not changed, as the Readme.md is not only displayed on github, but is also automatically processed and displayed in various places (e.g. the tabs for a product on the marketplace website, e.g. here: https://market.axonivy.com/a-trust). Could you please contact the developer in Team Atomic and ask for an adjustment of the documentation?

@andreasbalsiger FYI

**Comment by nntthuy-axonivy** [comment]
Hi @andreasbalsiger @ivy-sgi, Atomic team has adapted Readme file same as the template. Please have a look on this pull request of Approval Decision Utils. Thank you!

FYI @nvvi

File: approval-decision-utils-product/README.md [hunk 1]
```diff
@@ -0,0 +1,237 @@
+
+# Approval Decision Utils
+  
+This component provides a standardized approach for implementing the approval flow. Users can make decisions, confirmations, leave comments, and view the approval history in a data table.
```
Comment (andreasbalsiger) [inline approval-decision-utils-product/README.md:4]: Let's pimp this text a bit up (marketing):
Axon Ivy's **Approval Decision** Utils provides a standardized approach for implementing your approvals in any business process.  This component:

- Gives you a standard pattern to make decisions and confirmations
- Implements a comment function for better documentation
- Provides a clear view of the ongoing approval history to meet regulatory, compliance, and transparency requirements
- Supports you with an easy-to-copy demo implementation to reduce your integration effort.

File: approval-decision-utils-product/README.md [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+![](./images/4-request-history.PNG)
+
+###  4. Select checkbox confirmation
+![](./images/5-request-confirmation.PNG)
+
+## Setup
+  
+###  Approach  
+  
+Decision data is stored in database table named `ApprovalHistory`. This table stores the selected decision, comment, approval date and confirmations.
```
Comment (andreasbalsiger) [inline approval-decision-utils-product/README.md:24]: Just to be coherent:
Decision data is stored in a database table named `ApprovalHistory.` This table stores the selected decisions, comments, approval dates, and confirmations.

**Review by andreasbalsiger — COMMENTED** [review]
Andreas' marketing pimp your market item input :)

**Comment by nntthuy-axonivy** [comment]
Hi @andreasbalsiger , I've just pushed a new commit to incorporate the feedback you gave. Could you please review the revised README file again? Thank you!

FYI @nvvi

**Pull request merged**